### PR TITLE
[Gen 2][Quickstart] Add React Native (Expo)

### PR DESCRIPTION
**Pull request by salihgueler**
#### Description of changes:

Added react native to the example app.

Moved Flutter and React Native to first in the order of the platforms.

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ x Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/start/mobile-support/index.mdx [hunk 1]
```diff
@@ -258,257 +223,543 @@ export const data = defineData({
...
 
-With the click, we update the checkbox but with the long click we remove it. Now if you run the application you should see the following flow.
+### Create Amplify Project
 
-<video autoPlay={true} muted={true} loop={true} width="40%" playsInline={true}>
-  <source src="/images/gen2/getting-started/android/android-getting-started-2.mp4" />
+The easiest way to get started with AWS Amplify is through npm with `create-amplify` command. You can run it from your base project directory.
+
+```bash
+npm create amplify@beta
```
Comment (ErikCH) [inline src/pages/[platform]/start/mobile-support/index.mdx:446]: For release, this won't be `@beta` I don't think.

**Review by ErikCH — APPROVED** [review]
Just left one comment. Will need another reviewer that can look at RN code.

File: src/pages/[platform]/start/mobile-support/index.mdx (hunk 1 above)
Comment (salihgueler) [reply src/pages/[platform]/start/mobile-support/index.mdx:446]: This is targeting to improve support for current docs.

**Comment by salihgueler** [comment]
Will be done by @swaminator's quick start list.

**Pull request closed without merging**